index.js

```diff
@@ -92,11 +95,51 @@ getStdin().then(str => {
     ? policy.ports.range
     : null;
 
+  var maxSize = (policy.size)
+    ? policy.size.max
+    : null;
+
+  var warningSize = (policy.size)
+    ? policy.size.warning
+    : null;
+
   var failedEnvKeys = [];
   var failedLabels = [];
   var failedVolumes = false;
   var failedPortRequirement = false;
   var failedPortRange = false;
+  var failedMaxSize = false;
+  var failedWarningSize = false;
+
+  if (warningSize !== null && containerSize >= warningSize) {
+    failedWarningSize = true;
+  }
+
+  if (maxSize === null ) {
+
+    if (failedWarningSize) {
+      console.log("[%s] %dMB container size should be below %dMB", clc.yellow('WARN'), containerSize, warningSize);
```

Comment:
I don't understand why not just log right outside of `if (maxSize...)` instead of duplicating twice